tests/receipt/test_get.py

```diff
@@ -0,0 +1,288 @@
+"""
+Provide tests for command line interface's get receipts command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+TRANSACTION_IDENTIFIERS_PRESENTED_ON_THE_TEST_NODE = \
+    'e79a883581c184787360de8607c5f970cdeeaa684af3e50d8532aa9dd07afa8e' \
+    '7fc92f0dc509b41b9695e795704bdd50455bebd1ed327a5330710ba40698b492, ' \
+    '6593d21046519022ba32c98e934d7dfc81e8b4edf6c064dbf70feb13db431087' \
+    '3ec00816bce8660cafd4fa2a8c80d0147d63cf616c624babd03142c694272017'
+
+
+def test_get_receipts_with_identifiers():
+    """
+    Case: get a list of the transaction's receipts by identifiers.
+    Expect: a list of the transaction's receipts is returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'receipt',
+        'get',
+        '--ids',
+        TRANSACTION_IDENTIFIERS_PRESENTED_ON_THE_TEST_NODE,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    list_of_receipts = json.loads(result.output).get('result')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+
+    count = 0
+    for identifier in TRANSACTION_IDENTIFIERS_PRESENTED_ON_THE_TEST_NODE.split(','):
+        assert identifier.strip() == list_of_receipts[count].get('id')
+        count += 1
```

Comment:
Seems, you have a way to do it without count, just loop and checking by `in`.

Reply:
@dmytrostriletskyi, done.